test/cypress/integration/payouts/payouts-list.cy.js

```diff
@@ -34,7 +34,7 @@ describe('Payout list page', () => {
 
   it('should correctly display payouts given a successful response from Ledger', () => {
     const payouts = [
-      { gatewayAccountId: liveGatewayAccountId, paidOutDate: '2019-01-29T08:00:00.000000Z' }
+      { gatewayAccountId: liveGatewayAccountId, paidOutDate: '2024-11-27T15:34:00.000000Z' }
```

Comment:
To make sure we’re really outputting the year and not the week year, can we make this 2024-12-31, which is in week year 2025 in both the UK and in the US? (UK because we’re in the UK and US because we it’s a common unintentional default software often gets run in.)